src/marketplace/ShowTankPopup.js

```diff
@@ -61,4 +61,4 @@ class ShowTankPopup extends React.Component<Props, State> {
 	}
 }
 
-export default ShowTankPopup;
\ No newline at end of file
+export default ShowTankPopup;*/
```

Comment:
Why is this all commented out? This is baffling. If this file is not needed why keep it here?

Reply:
Sorry i thought i had deleted this file.